db/institutions.yml

```diff
@@ -262,4 +262,108 @@ nyu:
   name: New York University
   state: NY
   zip: 10003
+vanderbilt:
+  country: USA
+  aka:
+  - Vanderbilt
+  - vanderbilt
+  city: Nashville
+  name: Vanderbilt University
+  state: TN
+  zip: '37240'
+milwaukee:
+  country: USA
+  aka: ['UW Milwaukee']
+  city: Milwaukee
+  name: University of Wisconsin - Milwaukee
+  state: WI
+  zip: '53211'
+arizona_state:
+  country: USA
+  aka:
+  - Arizona State
+  - arizona state
+  city: Tempe
+  name: Arizona State University
+  state: AZ
+  zip: '85287'
+ohio:
```

Comment:
for these I have been adding u at the beginning and end, so ``columbiau``, this would be ``ohiou``.  I would like to keep it consistent.